lib/ansible/modules/system/java_cert.py

```diff
@@ -77,23 +82,29 @@
     keystore_path: /usr/lib/jvm/jre7/lib/security/cacerts
     keystore_pass: changeit
     state: present
-
 - name: Remove certificate with given alias from a keystore
   java_cert:
     cert_url: google.com
     keystore_path: /usr/lib/jvm/jre7/lib/security/cacerts
     keystore_pass: changeit
     executable: /usr/lib/jvm/jre7/bin/keytool
     state: absent
-
+- name: Import trusted CA from SSL certificate
```

Comment:
Blank line between examples please
```suggestion

- name: Import trusted CA from SSL certificate
```